lib/kitchen/driver/vagrant.rb

```diff
@@ -389,6 +402,12 @@ def update_state(state)
         state[:rdp_port] = hash["RDPPort"] if hash["RDPPort"]
       end
 
+      # @return [String] full absolute path to the kitchen cache directory
+      # @api private
+      def local_kitchen_cache
+        @local_kitchen_cache ||= File.expand_path('~/.kitchen/cache')
```

Comment:
I'm not sure this format will work on Windows. We might need to do something a little more Windows friendly?